docs/install_guides/installing-model-dev-tools.md

```diff
@@ -10,7 +10,7 @@ OpenVINO Development Tools is a set of utilities that make it easy to develop an
 
 The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.
 
-In both cases, Python 3.6 - 3.10 need be installed on your machine before starting.
+In both cases, Python 3.7 - 3.10 need be installed on your machine before starting.
```

Comment:
```suggestion
In both cases, Python 3.7 - 3.10 need to be installed on your machine before starting.
```

Reply:
I would even make it "needs to be" because just one version is required.